library/Elasticsearch/Elastic.php

```diff
@@ -39,8 +39,11 @@ public static function extractFields($source, &$fields, array $parent = [])
         foreach ($source as $key => $value) {
             if ($key === '@timestamp') {
                 $fields['@timestamp'] = function($event) {
+                    $tz=date_default_timezone_get()
```

Comment:
That's missing a closing semi-colon, isn't it?

Reply:
You're right, now it works! thank you!

Reply:
It was definitely missing a semi-colon.
Sorry for that.
Should use a git client instead of copy the code by hand ...